### PR TITLE
Cloud 458

### PR DESCRIPTION
**Pull request by lee-baines**
Moved Prometheus into forgeops from forgeops-dashboard.
Simplified deployment helm charts and deployment scripts so can be deployed by with out any flags to monitoring namespace.
Updated readme.

### CODE REVIEW

File: monitoring/README.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+# Prometheus and Grafana deployment
+
+This deployment uses the [CoreOS Prometheus Operator](https://coreos.com/operators/prometheus/docs/0.15.0/index.html). 
+
+**This monitoring folder contains the following functions:**
```
Comment (shankar-forgerock) [inline monitoring/README.md]: s/This/The

File: monitoring/README.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+* deploy scripts to:
+    * deploy the Prometheus Operator along with Grafana and Alert Manager and other Helm charts that help monitor GKE.
+    * connect to the Prometheus and Grafana endpoints.
+* exporter-forgerock Helm chart that contains a Helm chart that provides:
```
Comment (shankar-forgerock) [inline monitoring/README.md]: s/that contains a//

File: monitoring/README.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    * configurable ServiceMonitors that define ForgeRock product endpoints to be monitored by Prometheus.
```
Comment (shankar-forgerock) [inline monitoring/README.md]: May be better as: * exporter-forgerock Helm chart that provides configurable ServiceMonitors. The ServiceMonitorst define the ForgeRock Identity Platform component endpoints that are 
monitored by Prometheus.

File: monitoring/README.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+* values files that are used by the deploy script and can be edited to customize the configuration of Prometheus, Grafana and Alert Manager.
+
+<br />
+
+# How Prometheus works
+
+The Prometheus operator works by watching for ServiceMonitor CRDs (CRDs are Kubernetes Custom Resource Definitions). These are first
```
[the remark this answers is not in the document]
Comment (lee-baines) [reply monitoring/README.md]: yes it is, I'll change

File: monitoring/README.md (hunk 3 above)
Comment (lee-baines) [reply monitoring/README.md]: sounds good, changed

File: monitoring/README.md (hunk 1 above)
Comment (lee-baines) [reply monitoring/README.md]: done

**Comment by lee-baines** [comment]
I've changed the forgerock helm chart from exporter-forgerock to forgerock-servicemonitors as its not actually an exporter like other prometheus exporters. Also, the new name is more explicit to what it does. I've tested deployment and remove to ensure this hasn't broken anything

File: monitoring/README.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
+# Prometheus and Grafana deployment
+
+The deployment uses the [CoreOS Prometheus Operator](https://coreos.com/operators/prometheus/docs/0.15.0/index.html). 
+
+**The monitoring folder contains the following functions:**
```
Comment (shankar-forgerock) [inline monitoring/README.md]: s/functions/artifacts

File: monitoring/helm/custom.yaml [hunk 6]
```diff
@@ -0,0 +1,39 @@
+# This yaml file provides override values for the exporter-forgerock chart
```
Comment (dgoldssfo) [inline monitoring/helm/custom.yaml]: Periods at the end of sentences, please

File: monitoring/README.md (hunk 5 above)
Comment (lee-baines) [reply monitoring/README.md]: done

**Pull request merged**